aggregate/housing_security/nycha_tenants.py

```diff
@@ -0,0 +1,81 @@
+from aggregate.decennial_census.decennial_census_001020 import decennial_census_001020
+from aggregate.aggregation_helpers import order_aggregated_columns
+import pandas as pd
+from internal_review.set_internal_review_file import set_internal_review_files
+from utils.PUMA_helpers import clean_PUMAs, puma_to_borough
+dcp_pop_races = ["anh", "bnh", "hsp", "onh", "wnh"]
```

Comment:
can import this dictionary from the `utils/dcp_population_excel_helpers`

Reply:
which dictionary exactly?